modules/ding_adgangsplatformen/ding_adgangsplatformen.module

```diff
@@ -521,6 +547,34 @@ function ding_adgangsplatformen_has_token_client() {
   return $clientId && $clientSecret;
 }
 
+/**
+ * Revoke an OAuth2 access token using a provider.
+ *
+ * @param \League\OAuth2\Client\Provider\GenericProvider $provider
+ *   The OAuth2 provider to use when revoking the token.
+ * @param string $access_token
+ *   The access token to revoke.
+ *
+ * @return bool
+ *   Whether revoking the token succeeded or not.
+ */
+function ding_adgangsplatformen_revoke_token(GenericProvider $provider, $access_token) {
+  $request = $provider->getAuthenticatedRequest('DELETE',
+    'https://login.bib.dk/oauth/revoke', $access_token);
```

Comment:
Not super happy about the URL placement here. All the other URL's are kept together in the `ding_adgangsplatformen_get_configuration()`.

Reply:
Maybe a minor rewrite of `ding_adgangsplatformen_get_configuration()` is requires as I remember that the return value is pasted directly to the oAuth library, which is not super happy about changes to the config array.

Reply:
> All the other URL's are kept together in the ding_adgangsplatformen_get_configuration()

Apart from https://login.bib.dk/oauth/revoke in the current version, but OK - I will bite and see what can be done.

As I understand it [the OAuth library is looking for specific keys in the configuration](https://github.com/thephpleague/oauth2-client/blob/2.3.0/src/Provider/AbstractProvider.php#L95-L108). As long as we do not override these we should be fine. We already put in a lot of other options which are irrelevant to the provider.